README.md

```diff
@@ -158,131 +155,120 @@ The support level of each EOS SDK features as of the current release of the plug
 
-6. Add the scenes to the ```Scenes In Build``` section of the ```Build Settings``` window. This can be done quickly by using the ```Shift``` key to select each scene at the same time, then dragging them into the proper area. Repeating for each sample pack folder.
+5. In the `Project` window, navigate to the scenes folders containing their respective sample scenes. `\Assets\Samples\PlayEveryWare EOS Plugin for Unity\[Version #]\[Pack Name]\Scenes`.
 
-    > :heavy_exclamation_mark: If you have other scenes already, and plan to look at the samples in a build, drag a sample scene to be the 0th scene in Build Settings before you build.
+6. Add the scenes to the `Scenes In Build` section of the `Build Settings` window. This can be done quickly by using the `Shift` key to select each scene at the same time, then dragging them into the proper area. Repeating for each sample pack folder.
 
-<br />
+    > [!WARNING]
+    > If you have other scenes already, and plan to look at the samples in a build, drag a sample scene to be the 0th scene in Build Settings before you build.
 
 ## Running the samples
 
-> :heavy_exclamation_mark: The plugin must be <a href="#configuring-the-plugin">configured</a> for samples to be functional.
-
-> :heavy_exclamation_mark: Some Samples may not be accessible if the extra packs were not <a href="#importing-the-samples">imported</a>, or the scenes weren't added in the build settings. 
+> [!IMPORTANT]
+> The plugin must be <a href="#configuring-the-plugin">configured</a> for samples to be functional. Some Samples may not be accessible if the extra packs were not <a href="#importing-the-samples">imported</a>, or the scenes weren't added in the build settings. 
 
 Sample walkthroughs can be found [here](docs/Walkthrough.md).
 
 <details>
   <summary><b>Steps to run a sample in editor</b></summary>
 
-> :heavy_exclamation_mark: The Social Overlay Feature is not supported in editor.
+  > [!NOTE]
+  > The Social Overlay Feature is not supported in editor.
 
 1. In the Unity editor, open the desired sample scene from the imported Scenes folder.
 
 2. Press the play button at the top of the editor.
 
-3. Login with a selected authentication type. <br />```Account Portal``` and then ```PersistentAuth``` is easiest for the first time. ```Dev Auth``` can be used for quicker iteration, among other options described [here](docs/Walkthrough.md), to explore features that don't require an Epic Games Account, or the options to choose for closer representation to the user experience on specific platform releases.
+3. Login with a selected authentication type. <br />`Account Portal` and then `PersistentAuth` is easiest for the first time. `Dev Auth` can be used for quicker iteration, among other options described [here](docs/Walkthrough.md), to explore features that don't require an Epic Games Account, or the options to choose for closer representation to the user experience on specific platform releases.
```

Comment:
Can't quite get where the latter part of the sentence is going. Maybe splitting into its own explanation could be more expressive for users.

Reply:
While I don't disagree - if you look at the changes made - the changes are only to the number of tick marks used, I actually make no change to the language.

Reply:
Oh, just realized this PR is mainly style and formatting. sorry.
Then I think that line needs a task of its own, since the extra explanation seems redundant that there is a login type doc for that.